lesson/day03.md

```diff
@@ -329,17 +328,7 @@ that all data on the calculator, from numbers to text to pictures, is really
 just an endless series of numbers to the computer (in fact, it isn't even
 that), and programs are no exception. This is related to an important point of
 computer science, and I want you to make it your mantra: _"Data is whatever
-you define it to be"_.  
-Seriously, the computer is an idiot and doesn't know dick-all about what this
-byte or that word is supposed to mean, that's why you can force Microsoft Word
-to open a JPEG. It won't look like much, but the fact that you can do it
-proves a related point, and that is that no matter what you do with your data,
-be consistent with it. If in one moment you point your finger at a memory cell
```

Comment:
The point about consistency is important. Maybe keep a version of this but turn it into a footnote.